src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionFeaturesTests.cs

```diff
@@ -164,6 +169,88 @@ void M()
         AssertJsonEquals(resolvedItem.Documentation, expectedDocumentation);
     }
 
+    [Theory, CombinatorialData]
+    [WorkItem("https://github.com/dotnet/roslyn/issues/69576")]
+    public async Task TestImportCompletionForExtensionMethod(bool mutatingLspWorkspace)
+    {
+        var markup =
+@"
+namespace NS2
+{
+    public static class ExtensionClass
+    {
+        public static bool ExtensionMethod(this object o) => true;
+    }
+}
+
+namespace NS1
+{
+    class Program
+    {
+        void M(object o)
+        {
+            o.{|caret:|}
+        }
+    }
+}";
+        await using var testLspServer = await CreateTestLspServerAsync(markup, mutatingLspWorkspace, DefaultClientCapabilities);
+        var completionParams = CreateCompletionParams(
+            testLspServer.GetLocations("caret").Single(),
+            invokeKind: LSP.VSInternalCompletionInvokeKind.Explicit,
+            triggerCharacter: "\0",
+            triggerKind: LSP.CompletionTriggerKind.Invoked);
+
+        // Make sure the import completion option is on.
+        testLspServer.TestWorkspace.GlobalOptions.SetGlobalOption(CompletionOptionsStorage.ShowItemsFromUnimportedNamespaces, LanguageNames.CSharp, true);
```

Comment:
do we need to ensure these options are also turned on in vscode?  Or is it covered by 
<img width="381" alt="image" src="https://github.com/dotnet/roslyn/assets/5749229/491aa1d3-f7dd-425f-9952-14f410b18aad">

Reply:
Yea, they are the same https://github.com/dotnet/vscode-csharp/blob/b5f8db1c1b890a7d6c6412e27216ac8e9752f4fe/package.json#L1210